snare/cloner.py

```diff
@@ -299,6 +294,13 @@ async def run(self):
     async def close(self):
         if not self.runner:
             raise Exception("Error initializing cloner!")
```

Comment:
I just spot we have initialization error in close function 🤔

Reply:
Strangely enough, that check is there just to be thorough and satisfy linter warnings. If you are referring to an instance where the error was actually raised, could you please share the steps to replicate it? I've tried it with multiple websites and it seems to work fine.